src/main/hook/patch/gfx.c

```diff
@@ -170,3 +215,7 @@ void patch_gfx_scale(enum patch_gfx_scale_mode scale_mode)
 {
   log_error("GFX scaling enabled but will not work. Temporarily removed due to issues with loading libGL.so");
 }
+
+void patch_gfx_frame_limit(uint16_t frame_limit){
+  s_frame_limit = frame_limit;
```

Comment:
```suggestion
  log_info("Enabled frame limit: %d frames", frame_limit);

  s_frame_limit = frame_limit;
```

Creates visibility in the log output which is useful when people report issues.